src/main/java/gov/nasa/pds/api/engineering/controllers/MyCollectionsApiController.java

```diff
@@ -116,6 +119,8 @@ private Products getProductChildren(String lidvid, int start, int limit, List<St
         RestHighLevelClient restHighLevelClient = this.esRegistryConnection.getRestHighLevelClient();
          
     	try {
+    		if (!lidvid.contains("::") && !lidvid.endsWith(":")) lidvid = this.getLatestLidVidFromLid(lidvid);
```

Comment:
Yes. Otherwise lid: or lid:: will find the latest lid::vid. You specifically asked that lid: return with a 404.  From your acceptance criteria at top of issue #4 

> http://localhost:8080/collections/urn:nasa:pds:izenberg_pdart14_meap:data_imagecube:: --> 404 (not found), lidvid resolution case

If the acceptance criteria have changed, that is fine too but need to be clear.